src/functionaltests/resources/features/threadingAndWaitlistRepeat.feature

```diff
@@ -0,0 +1,29 @@
+#Author: your.email@your.domain.com
+#Keywords Summary :
+#Feature: List of scenarios.
+#Scenario: Business rule through list of steps with arguments.
+#Given: Some precondition step
+#When: Some key actions
+#Then: To observe outcomes or validation
+#And,But: To enumerate more Given,When,Then steps
+#Scenario Outline: List of steps for data-driven as an Examples and <placeholder>
+#Examples: Container for s table
+#Background: List of steps run before each of the scenarios
+#""" (Doc Strings)
+#| (Data Tables)
+#@ (Tags/Labels):To group Scenarios
+#<> (placeholder)
+#""
+## (Comments)
+#Sample Feature Definition Template
+@SubscriptionTriggerFeature
+Feature: Subscription trigger test
+
+  @ThreadingAndWaitlistRepeatScenario
+  Scenario: Test multithreaded events processing, waitlist resend and waitlist TTL.
+  	Given that eiffel events are sent
+	And no event is aggregated

```

Comment:
add step **_no event is aggregated_** after step **_waitlist should not be empty_**